### ISSUE TITLE
Pickers: Remove button present next to colors is not visible when focus goes on them in high contrast mode white and black. (Bug 6)

### ISSUE THREAD

**Issue by jdhuntington** [OfficeDev/office-ui-fabric-react#10825]
Breakout of issue #8655

**Comment by khmakoto** [comment]
This issue does not reproduce anymore as can be seen in the screenshots below:

### High Contrast Black

__Before:__
![image](https://user-images.githubusercontent.com/7798177/67315977-80983b00-f4bc-11e9-923c-3c34ffb4923c.png)

__After:__
![image](https://user-images.githubusercontent.com/7798177/67315965-7c6c1d80-f4bc-11e9-8e39-4917be6d0dde.png)

### High Contrast White

__Before:__
![image](https://user-images.githubusercontent.com/7798177/67316043-9c9bdc80-f4bc-11e9-9d9d-538af5c06d8d.png)

__After:__
![image](https://user-images.githubusercontent.com/7798177/67316113-bb01d800-f4bc-11e9-8c18-35431288530d.png)

Given that, I'll be closing this issue accordingly.
